android-components/components/feature/prompts/src/main/res/values/strings.xml

```diff
@@ -130,14 +130,18 @@
     <string name="mozac_feature_prompts_manage_address">Manage addresses</string>
 
     <!-- Federated Credential Management prompts -->
+    <!--Content description for the Provider favicon in the Select Provider FedCM prompt -->
+    <string name="mozac_feature_prompts_provider_favicon">Provider favicon</string>
     <!--Content description for the Account picture in the Select Account FedCM prompt -->
     <string name="mozac_feature_prompts_account_picture">Account picture</string>
     <!-- Title of the Identity Credential provider dialog choose. -->
     <string name="mozac_feature_prompts_identity_credentials_choose_provider">Choose a login provider</string>
-    <!-- Title of an account picker dialog for identity credentials. -->
-    <string name="mozac_feature_prompts_identity_credentials_choose_account">Choose an account</string>
+    <!-- Title of an account picker dialog for identity credentials. The %1$s will be replaced with the name of the provider -->
+    <string name="mozac_feature_prompts_identity_credentials_choose_account_for_provider">Sign in with a %1$s account</string>
```

Comment:
we have a process to update strings [documented here](https://github.com/mozilla-mobile/fenix/blob/main/docs/Working-with-Strings.md). The page is technically for Fenix, but we likely follow a similar process here for ensuring any string updates get flagged for re-localization.